LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1888,15 +1888,15 @@ http/tests/webgpu/webgpu/api/operation/compute_pipeline/overrides.html [ Pass ]
 http/tests/webgpu/webgpu/api/operation/device/lost.html [ Pass ]
 http/tests/webgpu/webgpu/api/operation/rendering/stencil.html [ Pass ]
 http/tests/webgpu/webgpu/api/operation/rendering/indirect_draw.html [ Pass ]
-http/tests/webgpu/webgpu/api/operation/reflection.html [ Pass ]
+http/tests/webgpu/webgpu/api/operation/reflection.html [ Skip ]
```

Comment:
new tests were added, which we don't currently pass, so add Skip expectations for now